src/host/outputStream.cpp

```diff
@@ -344,7 +344,12 @@ bool ConhostInternalGetSet::PrivateShowCursor(const bool show) noexcept
 bool ConhostInternalGetSet::PrivateAllowCursorBlinking(const bool fEnable)
 {
     DoSrvPrivateAllowCursorBlinking(_io.GetActiveOutputBuffer(), fEnable);
-    return true;
+
+    bool isPty;
+    DoSrvIsConsolePty(isPty);
+    // If we are connected to a pty, return that we could not handle this

```

Comment:
Confusing. We did handle it by turning on our own blinking... didn't we with `DoSrvPrivateAllowCursorBlinking` above?

Reply:
Returning `false` in an event handler for the VT Parser's OutputStateMachineEngine has been overloaded to mean, "Pass this through to the connected terminal."

This is not the first, and will likely not be the last, case in which we do this.

Reply:
So we are saying that we need to process it AND the connected terminal needs to process it too... right?

Reply:
So technically we don't _need_ to process it ourselves, only the connected terminal needs to. I've done it this way just so there's consistency between the two, just in case.

Reply:
Just leave that as a comment then to inform a future reader, please, and I'm good then.

Reply:
Okay, added!